packages/core/src/crypto/jose/jwk/ecCompression.ts

```diff
@@ -43,6 +44,21 @@ function getConstantsForCurve(curve: 'P-256' | 'P-384' | 'P-521') {
     pIdent = prime.add(1).divide(4)
   }
 
+  if (curve === 'secp256k1') {
```

Comment:
Curious, where did you find this? Can you add a link?

Reply:
Added a link [https://en.bitcoin.it/wiki/Secp256k1](https://en.bitcoin.it/wiki/Secp256k1)